public/app/plugins/datasource/elasticsearch/specs/query_builder.test.ts

```diff
@@ -65,8 +65,7 @@ describe('ElasticQueryBuilder', () => {
             metrics: [{ type: 'avg', field: '@value', id: '1' }],
             bucketAggs: [{ type: 'date_histogram', field: '@timestamp', id: '2' }],
           },
-          100,
```

Comment:
all of these changes are related to the removal of the last parameter in the `build` method